packages/model-viewer/src/three-components/ModelScene.ts

```diff
@@ -673,11 +674,43 @@ export class ModelScene extends Scene {
       return null;
     }
 
+    if (hit.uv == null) {
+      return null;
```

Comment:
I think we don't want  this; a model can lack UV coordinates, in which case `uv = null` makes sense, and it shouldn't destroy the rest of the result.